backend/db_funcs/hintFuncs.py

```diff
@@ -41,14 +41,14 @@ def deleteHint(hint_id: int):
     return {"result": "removed"}
 
 
-def getHint(hint_id: int):
+def get_hint(hint_id: int):
     try:
         with DB_conn.getConn(connection):
             with DB_conn.getCursor(connection) as cur:
                 cur.execute("SELECT * FROM hint WHERE hint_id = %s;", (hint_id,))
-                result = cur.fetchall()
+                result = cur.fetchall()[0]
     except:
-        return {"result": "error"}
+        return None
     return result
```

Comment:
úplně bych odmazal ten try: except, není to potřeba. Chyby v databázi ať klidně vyhodí API chybu. Takže budem vracet jen result. Navíc prosím nevracet objekt z databáze, ale vracet pythonní objekt - podívej se na to, jak to řeším u `people`/`user`, respektive jak jsem ti to upravil u `cipherGame`